launch.go

```diff
@@ -387,7 +387,7 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 	// Add coverage env vars
 	coverageInfo, err := api.GetCoverageInfo()
 	if err != nil {
-		return fmt.Errorf("Getting coverage info for build %v", build.ID)
+		log.Printf("Failed to get coverage info for build %v so skip it\n", build.ID)
 	}
 
 	for key, value := range coverageInfo.EnvVars {
```

Comment:
We should not execute this `for` loop, if an error occurs.

Reply:
fixed!

Reply:
coverageInfo will be empty map object according to the type of the func, so this works as a zero-length loop.
https://github.com/screwdriver-cd/launcher/blob/master/screwdriver/screwdriver.go#L298

Reply:
yes, I think it's tested by added test case. But not executing is looks better.